optuna_dashboard/artifact/_backend.py

```diff
@@ -163,6 +196,25 @@ def delete_artifact(study_id: int, trial_id: int, artifact_id: str) -> dict[str,
         response.status = 204
         return {}
 
+    @app.delete("/api/artifacts/<study_id:int>/<artifact_id:re:[0-9a-fA-F-]+>")
+    @json_api_view
+    def delete_study_artifact(study_id: int, artifact_id: str) -> dict[str, Any]:
+        if artifact_store is None:
+            response.status = 400  # Bad Request
+            return {"reason": "Cannot access to the artifacts."}
+        artifact_store.remove(artifact_id)
+
+        # The artifact's metadata is stored in one of the following two locations:
```

Comment:
[question] As study artifact is introduced in this PR, the metadata is always stored in `ARTIFACTS_ATTR_PREFIX + artifact_id`?

Reply:
Yes, this behavior is the same as [optuna/artifacts/_upload.py](https://github.com/optuna/optuna/blob/f966055e159bd16093b679e7c3c316c9d2c0d2b5/optuna/artifacts/_upload.py#L82)

Reply:
In that case, can we remove those lines?

```
storage.set_study_system_attr(
            study_id, _dashboard_artifact_prefix(study_id) + artifact_id, json.dumps(None)
        )
```

Reply:
It sounds good. I think I can remove that line from `delete_study_artifact`. However, I would not remove it from `delete_trial_artifact` due to backward compatibility.